src/server/command.ts

```diff
@@ -68,13 +68,18 @@ function sshOptions(
     port,
     '-o',
     `PreferredAuthentications=${auth}`,
-  ];
+    ];
+  console.log(`Authentication Type: ${auth}`)  
```

Comment:
please dont use `console.log` import the logger and use info logging

Reply:
Fixed in latest commit